app/javascript/utils/donations.js

```diff
@@ -107,10 +107,13 @@ $(function() {
     }
   )
 
+  // As of #3373, item quantity field is a string rather than numeric input type
+  // (due to cross browser inconsistencies). Therefore it needs to be converted
+  // to a number for the large donation check.
```

Comment:
I don't think this comment is really needed... someone looking at this code right now would never know the input was ever a numeric input in the first place.

Reply:
Removed.